### DISCUSSION TITLE
Copilot wont work in certain directories

### DISCUSSION THREAD

**Discussion by jason-scheffel** [community/community#16592] [Copilot Conversations]
Hello, when I try to use copilot in any file inside of my home directory or any directory on my ssd where copilot and the below are installed, it works perfectly. However, when I try to use it on my hdd which contains only files such as python scripts, it does not work; when I run Copilot status, it says everything is okay too; however, there are no suggestions. 

I have also tried restarting the system, restarting copilot, and logging in and out.

I have also tried the same using Visual Studio Code and have gotten the same result as above; moreover, I have tried it on a fresh install of my system and it is the same.

I have installed the latest version for the plugin for nvim; also, here are other packages versions:

- nvim - 0.7.0
- node - 18.1.0

I am also using the latest version of Endeavor Os, which is based on Arch.

edit:
I have also tried to make it work by installing through packer and installing directly and it is the same as above. Furthermore, the same is above for regular neovim or ide layers such as lunarvim

**Comment by coalio** [comment]
I have this same issue. However, it seems that Copilot crashes under paths containing certain symbols such as `#`. 

I've successfully replicated this issue with the VSCode Copilot plugin by testing several files containing `#` at some part of the path. Copilot failed to work in any of them and would only work if I renamed the conflicting directory.

**Reply by jason-scheffel** [reply]
I created a directory on my system containing a # and a file containing a #, the directory is the following:

`/run/media/myname/HDD/Q/test/fsdff#`

and the file is called:

`test#.py`

Copilot was able to work for me with nodejs-lts-erbium installed instead of the regular nodejs package, so mabey that is why.

**Comment by jason-scheffel** [comment]
So for one solution is that i installed the following package instead of the latter package:
- nodejs-lts-erbium
- nodejs

This fixed it, though it is obviously unclear what the issue was specifically.

Moreover, I was able to test the original issue on arco linux and plain vanilla arch. The original issue persisted; however, installing that lts package seemed to fix it; at least on endeavor os--haven't reinstalled the other ones to test it.
